hw/ip/otbn/dv/model/iss_wrapper.cc

```diff
@@ -189,6 +189,37 @@ static uint32_t read_hex_32(const char *str) {
   return strtoul(buf, nullptr, 16);
 }
 
+static uint32_t read_ext_reg(const std::string &reg_name,
+                             const std::vector<std::string> &lines,
+                             uint32_t default_val) {
+  // We're interested in lines that show an update to the external register
+  // called reg_name. These look something like this:
+  //
+  //   otbn.$REG_NAME &= ~ 0x000001 (from HW) (now 0x000000)
+  //
+  // The \n picks up the newline that we expect at the end of each line.
+  std::regex re("\\s*otbn\\." + reg_name + ".*0x([0-9a-f]{8})\\)\n");
+  std::smatch match;
+
+  uint32_t val = default_val;
+
+  std::cerr << "read_ext_reg(" << reg_name << ", ...)\n";
```

Comment:
Is this leftover debug output (also the other std::cerr lines in this function)?

Reply:
Oops! Thanks!